README.md

```diff
@@ -272,8 +272,30 @@ Router.map(function() {
 }
 ```
 
-The route to transition to if the session is not authenticated can also be
-[configured](#configuration) to be another one than `login`.
+The route to transition to if the session is not authenticated should be set in each 
+route like: 
+
+```js
+// app/route/fun-place.js
+export default Route.extend(AuthenticatedRouteMixin, {
+  authenticationRoute: 'some-route-other-than-login'
+});
+```
```

Comment:
I think everything below this isn't something for the (already really long) README but should go into a guide instead.